cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/b2c/IssueCredentialFlowB2C.kt

```diff
@@ -23,29 +26,22 @@ object IssueCredentialFlowB2C {
     @InitiatingFlow
     @StartableByRPC
     open class Issuer(
```

Comment:
Why do we keep b2b\b2c separation?
IMHO, that we can create corda based agent interface and then join flows

Reply:
I'll create another ticket for this.

Reply:
https://luxproject.luxoft.com/jira/browse/CTO-187